base/etc/nginx/conf.d/header/robots.conf

```diff
@@ -0,0 +1 @@
+add_header X-Robots-Tag $header_x_robots_tag always;
```

Comment:
how does this work exactly?

Reply:
my expectation was we are adding this header conditionally to a response only on certain paths?

Reply:
If `$header_x_robots_tag` is empty then the header does not get set.

This value will only bet set on specific paths:

https://github.com/skpr/image-nginx/pull/35/files#diff-ddd1b71a5ef0184511530414aead69e0335f5914c978672686fb64a6180dbe5aR17

Reply:
I mean, so the value of $request_uri matches the first regex in `drupal/etc/nginx/robots.conf`, then uses the value on the right side, I guess

Reply:
Yep. Exactly how it works.